### PR TITLE
updated otsql dependency

### PR DESCRIPTION
**Pull request by ancarebeca**
**Why?**
Updated dependencies, before it was using `github.com/j2gg0s/otsql` now it is using `github.com/90POE/otsql`

### CODE REVIEW

File: example/base.go [hunk 1]
```diff
@@ -10,10 +10,10 @@ import (
 	"strings"
 	"time"
 
-	"github.com/j2gg0s/otsql"
-	"github.com/j2gg0s/otsql/hook/log"
-	"github.com/j2gg0s/otsql/hook/metric"
-	"github.com/j2gg0s/otsql/hook/trace"
+	"github.com/90POE/otsql"
```
Comment (arxdsilva) [inline example/base.go]: shouldnt it be 90poe?

File: example/github.com/uptrace/bun/go.mod [hunk 2]
```diff
@@ -1,10 +1,10 @@
-module github.com/j2gg0s/otsql/example/github.com/uptrace/bun
+module github.com/90POE/otsql/example/github.com/uptrace/bun
```
Comment (arxdsilva) [inline example/github.com/uptrace/bun/go.mod]: why `.../example/...` ? as we're kind of making this our own we can also change this

File: example/base.go (hunk 1 above)
Comment (ancarebeca) [reply example/base.go]: My bad, sorry, fixed [c439875](https://github.com/90poe/otsql/pull/2/commits/c43987543d7af447ada6f72537d76329ff3cac88)

File: example/github.com/go-sql-driver/mysql/go.mod [hunk 3]
```diff
@@ -1,12 +1,12 @@
-module github.com/j2gg0s/otsql/example/github.com/go-sql-driver/mysql
+module github.com/90poe/otsql/example/github.com/go-sql-driver/mysql
 
 go 1.17
 
-replace github.com/j2gg0s/otsql => ../../../../
+replace github.com/90poe/otsql => ../../../../
```
Comment (vyskocilm) [inline example/github.com/go-sql-driver/mysql/go.mod:6]: ~I am pretty sure that was not meant to be committed :-)~

Edit: my bad, /i missed the context.

**Review by vyskocilm — APPROVED** [review]

**Pull request merged**